### PR TITLE
Set batch transaction list size limit

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Set batch transaction list size limit
- Fix transaction bytes limit application 

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/327

### CODE REVIEW

File: integration/node.test.js [hunk 1]
```diff
@@ -2412,70 +2412,143 @@ describe('Blockchain Node', () => {
...
+        })
+      })
+
+      it('rejects a batch transaction carrying an over-size transaction.', () => {
+        const account = ainUtil.createAccount();
+        const client = jayson.client.http(server1 + '/json-rpc');
+        let longText = '';
+        for (let i = 0; i < TX_BYTES_LIMIT / 2; i++) {
+          longText += 'a'
+        }
```
Comment (cshcomcom) [inline integration/node.test.js:2525]: minor: How about `const longText = 'a'.repeat(TX_BYTES_LIMIT / 2);`?

**Review by cshcomcom — APPROVED** [review]
LGTM with a minor comment! Thanks

**Review by liayoo — COMMENTED** [review]
Should we perform the same checks at /batch client api?

File: integration/node.test.js (hunk 1 above)
Comment (platfowner) [reply integration/node.test.js]: Done. Thanks.

**Comment by platfowner** [comment]
@liayoo For the client APIs, I think it's fine as they're dev / debug APIs and will be disabled when we launch the mainnet.

**Comment by platfowner** [comment]
Thanks for the fast review. Now merging..

**Pull request merged**